src/docs/Contributing/SubmitChanges/SubmitNewPattern/index.mdx

```diff
@@ -3,28 +3,30 @@ title: 'Submit a new pattern'
 order: 10
 ---
 
-> If you are a designer you also need to read the [Definition of Done for Sketch symbols contribution](/Contributing/SubmitChanges/DesignToolsConventions/#sketch) page
+<Highlight theme="tips" title="Good to know">
 
-<br/>
+If you are a designer you also need to read the [Definition of Done for Sketch symbols contribution](Contributing/Designers/DesignToolsConventions) page
```

Comment:
link doesn't work for me

Reply:
@caqueste I think the good link it's => /Contributing/Designers/DesignToolsConventions/